scenes/globals/pause/pause_overlay.gd

```diff
@@ -38,3 +40,10 @@ func _on_options_back() -> void:
 
 func _on_resume_button_pressed() -> void:
 	toggle_pause()
+
+
+func _on_title_screen_button_pressed() -> void:
+	toggle_pause()
+	SceneSwitcher.change_to_packed_with_transition(
+		TITLE_SCENE, ^"", Transition.Effect.FADE, Transition.Effect.FADE
```

Comment:
FWIW I expected the button to take you back to Fray's End (and only be visible if you're not in Fray's End) rather than to the title screen. But this would also work!

Reply:
Oh I see, yes both can work. I think going back to the title screen would be better if we ultimately persist state, and for having the "credits" and "exit game" options reachable.

Reply:
I see - so in the case where you have some state saved, you can Continue and not play the intro again